src/matches.js

```diff
@@ -1,8 +1,10 @@
-function matches(textToMatch, node, matcher) {
+function fuzzyMatches(textToMatch, node, matcher, collapseWhitespace = true) {
```

Comment:
Perhaps for both of these the forth arg should be an options argument. This would have the benefit of being named so we don't need the constant in the queries file and also if we decide to allow forget configuration we could extend this with more options without trouble.

In addition, I wonder if we should allow users to configure this in their own calls to the queries by forwarding the options they pass to this 🤔 would that complicate things too much? If so, let's do it later if at all.

Reply:
Ok, implemented. I also separated `trim` and `collapseWhitespace` - everything gets trimmed by default but only content is collapsed.